library.gradle

```diff
@@ -26,11 +26,13 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles "consumer-rules.pro"
     }
+
     sourceSets {
-        main.java.srcDirs += 'src/main/kotlin'
-        main.java.srcDirs += 'src/main/java'
-        test.java.srcDirs += 'src/test/java'
+        unityIAP {
+            java.srcDirs = ['src/latestDependencies/java']
+        }
```

Comment:
Have you tested the docs generation after this change?

Reply:
ooo good call, no i haven't...will try that out

Reply:
was able to successfully run `dokkaHtmlMultiModule`...